pkgs/misc/stacki3/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, poetry2nix, fetchFromGitHub }:
+
+poetry2nix.mkPoetryApplication rec {
+  pname = "stacki3";
+  version = "1.0.0";
+
+  projectDir = fetchFromGitHub {
+    owner = "ViliamV";
+    repo = pname;
+    rev = "8b835f7caed5f10b7cf18f4ee50d7b9695f548e8";
+    hash = "sha256-MzU5YOhL2PvBFAZVfXYQbXAZ/PhOUQg1lX/qnPo/mqY=";
+  };
+
+  pyproject  = ./pyproject.toml;
```

Comment:
I know nothing about pyproject, but couldn't this be done by referencing the toml file that's in the src?